pay-api/migrations/versions/2024_07_12_f921d5e32835_add_expense_authority_email.py

```diff
@@ -22,6 +22,7 @@ def upgrade():
                     sa.Column('first_name', sa.String(25), nullable=True),
                     sa.Column('last_name', sa.String(25), nullable=True),
                     sa.Column('email', sa.String(25), nullable=False),
+                    sa.PrimaryKeyConstraint('id')
```

Comment:
This migration would have run already. I think you will need a new one to make changes to add a PK constraint to id.

Reply:
fixed